src/BlueOak-1.0.0.xml

```diff
@@ -17,15 +17,15 @@
       <p><optional>## </optional>Copyright</p>
       <p>Each contributor licenses you to do everything with this software that would otherwise infringe that contributor's copyright in it.</p>
       <p><optional>## </optional>Notices</p>
-      <p>You must ensure that everyone who gets a copy of any part of this software from you, with or without changes, also gets the text of this license or a link to <optional>&lt;</optional>https://blueoakcouncil.org/license/1.0.0<optional>&gt;</optional>.</p>
+      <p>You must ensure that everyone who gets a copy of any part of this software from you, with or without changes, also gets the text of this license or a link to &lt;https://blueoakcouncil.org/license/1.0.0&gt;.</p>
```

Comment:
The angle brackets here are nonsemantic. Blue Oak is still Blue Oak without them. Moreover, many Markdown renders will autodetect the URI and render as a hyperlink either way.

Reply:
Agreed -- I think that's fine, and I think omitting the `optional` tags here doesn't change that. My understanding of the SPDX matching guidelines is that angle brackets would be interpreted as optional in any event, as I don't see `optional` tags used for angle brackets in similar places in other license texts (e.g. [AGPL-3.0-only](https://github.com/spdx/license-list-XML/blob/master/src/AGPL-3.0-only.xml), [WTFPL](https://github.com/spdx/license-list-XML/blob/master/src/WTFPL.xml) as a couple random examples).

That said -- primarily I wanted to get the tests passing so that we could merge it and get it added. If you want to iterate on the markup for things like optional tags once it's added, that's fine, it isn't locked in stone for minor markup edits like that. In any event, it won't be appearing on the main license list at https://spdx.org/licenses until the next release, so we can continue to iterate. Either way is fine by me.

Reply:
Don't let me change the process. I am trying to learn *your* process. It's all fine by me!

Reply:
@swinslow where in the matching guidelines are the angle brackets described as optional?
I believe that other licenses do not have the `<optional>` tag simply because they haven't thought of it: they have selected one way (with or without the angle brackets) and any instance has to match -- and it usually does, since the text is copied.

Reply:
@zvr Quite possibly it's my own wrong assumptions...

I'd understood this to be the case, but looking at the [matching guidelines](https://spdx.org/spdx-license-list/matching-guidelines) now, I can't point to a specific rule that says so. I don't think this falls into punctuation which is included per rule 5.

Would you be comfortable with us merging this as-is currently, and perhaps we open a separate issue to address this across licenses?

Reply:
We could merge either of the two versions (with or without the angle brackets) and hope that the people who use it match our choice.
Since the origin URL has a markdown version *with* brackets, I'd rather we also use this one.

Once we manage to fix the issue with optional characters without spaces, we can re-visit for an optimization...